tests/backend_test_base.py

```diff
@@ -27,12 +27,15 @@ class Tf2OnnxBackendTestBase(unittest.TestCase):
     TMPPATH = tempfile.mkdtemp()
     BACKEND = "onnxruntime"
     OPSET = 7
+    IS_OPSET_SET = False
     DEBUG = None
 
     def debug_mode(self):
         return type(self).DEBUG
 
     def setUp(self):
+        if not self.IS_OPSET_SET and "OPSET_VERSION" in os.environ:
```

Comment:
we should either use environment variables or command line arguments.
What is wrong with the existing command line arguments?

Reply:
same as guoliang's commit-65fe0338f05
two ways to sest opset version, and setting at command line can override setting at env.